datafusion/sqllogictest/test_files/dictionary.slt

```diff
@@ -280,3 +280,70 @@ ORDER BY
 2023-12-20T01:20:00 1000 f2 foo
 2023-12-20T01:30:00 1000 f1 32.0
 2023-12-20T01:30:00 1000 f2 foo
+
+# Cleanup
+statement error DataFusion error: Execution error: Table 'm1' doesn't exist\.
```

Comment:
why is it not "ok", but an error?

Reply:
Good call -- this is a mistake and I will fix it